_meta/ecs-migration.yml

```diff
@@ -0,0 +1,154 @@
+# The ECS migration file contains the information about all the fields which are migrated to ECS in 7.0.
+# The goal of the file is to potentially have scripts on top of this information to convert visualisations and templates
+# based on this information in an automated way and to keep track of all changes which were applied.
+#
+# The format of the file is as following:
+#
+# - from: source-field-in-6.x
+#   to: target-filed-in-ECS
+#   # Alias field is useful for fields where there is a 1-1 mapping from old to new
+#   alias: true-if-alias-is-required-in-6x (default is true)
+#   # Copy to is useful for fields where multiple fields map to the same ECS field
+#   copy_to: true-if-field-should-be-copied-to-target-in-6x (default is false)
+
+- from: beat.hostname
+  to: agent.server.host.name
+
+- from: beat.name
+  to: agent.server.type
+
+- from: beat.version
+  to: agent.server.version
+
+- from: meta.cloud.availability_zone
+  to: cloud.availability_zone
+
+- from: meta.cloud.instance_id
+  to: cloud.instance.id
+
+- from: meta.cloud.instance_name
+  to: cloud.instance.name
+
+- from: meta.cloud.machine_type
+  to: cloud.machine.type
+
+- from: meta.cloud.project_id
+  to: cloud.project.id
+
+- from: meta.cloud.provider
+  to: cloud.provider
+
+- from: meta.cloud.region
+  to: cloud.region
+
+- from: context.service.agent.name
+  to: agent.name
+
+- from: context.service.agent.version
+  to: agent.version
+
+- from: context.system.architecture
+  to: host.architecture
+
+- from: context.system.ip
+  to: host.ip
+
+- from: context.system.platform
+  to: host.os.platform
+
+- from: context.request.method
+  to: http.method
+
+- from: context.request.http_version
+  to: http.version
```

Comment:
From non indexed fields we are also missing: 

- `context.response.headers.content-type`, `context.response.headers_sent`, `context.response.status_code`
- `context.request.headers.cookie`, `context.request.headers.user-agent`

One possibility would be to add `http.request` and `http.response` and then 
map to 
- `http.response.headers.content-type` ,`http.response.headers.sent`, `http.response.status_code`
- `http.request.method`, `http.request.headers.cookie`, `http.request.headers.user-agent` 

But there are also other fields in `context.request` and `context.response` that don't necessarily describe http requests. When splitting those fields up we loose some semantic grouping.

Reply:
Suggestion in graphaelli#1